src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
```

Comment:
The methods could benefit from input validation to ensure that 'a' and 'b' are of compatible types.